Documentation/Contributing/development-flow.md

```diff
@@ -207,6 +207,11 @@ All pull requests must pass all continuous integration (CI) tests before they ca
 automatically run against every pull request. The results of these tests along with code review feedback determine whether
 your request will be merged.
 
+For developers who would like
+to test their changes as much as possible locally before pushing
+(a behavior that is not required but highly encouraged), the unit tests and various linters can be run locally in a developer environment.
+ways to run unit tests and various linters locally in a developer environment.
```

Comment:
Remove this line since it was merged with the previous line (and maybe change this whole paragraph so the lines are not so different in length)
```suggestion
```

Reply:
?

Reply:
revised to have line lengths more similar and removed redundancy from the last line.